src/upnp_xml.cc

```diff
@@ -105,6 +105,9 @@ void UpnpXMLBuilder::renderObject(const std::shared_ptr<CdsObject>& obj, size_t
     if (obj->isItem()) {
         auto item = std::static_pointer_cast<CdsItem>(obj);
 
+        if (quirks != NULL)
```

Comment:
Probably should be `nullptr`?

Reply:
Resolved in https://github.com/gerbera/gerbera/pull/1288/commits/e23aeff8f0cdb93e062b27881e206442902dbba2